### PR TITLE
Happy number

### PR DESCRIPTION
**Pull request by salonishah01**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Happy Number code in python

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1308

### CODE REVIEW

File: Python/README.md [hunk 1]
```diff
@@ -209,6 +209,8 @@ _add list here_
 - [Reverse a string using stack](other/Reverse_string_using_stack.py)
 - [Russian Peasant Algorithm](other/Russian_Peasant_Algorithm.py)
 - [Satellite Image analysis using Numpy](other/Satellite%20Image%20Analysis%20using%20numpy.ipynb)
+- [Happy Number](other/HappyNumber.py)
+
```
Comment (atarax665) [inline Python/README.md]: Remove this blank line.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/README.md (hunk 1 above)
Comment (salonishah01) [reply Python/README.md]: @atarax665 Done.

**Review by atarax665 — APPROVED** [review]
LGTM :)

**Comment by atarax665** [comment]
Resolve merge conflict.

**Review by iamrajiv — APPROVED** [review]

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

**Pull request merged**